src/WebApp/AIOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace eShop.WebApp;
+
+public class AIOptions
+{
+    /// <summary>Settings related to the use of OpenAI.</summary>
+    public OpenAIOptions OpenAI { get; set; } = new();
+}
+
+public class OpenAIOptions
+{
+    /// <summary>OpenAI API key for accessing embedding LLM.</summary>
+    public string? ApiKey { get; set; }
+
+    /// <summary>Optional endpoint for which OpenAI API to access.</summary>
+    public string? Endpoint { get; set; }
+
+    /// <summary>The name of the chat model to use.</summary>
+    public string ChatName { get; set; } = "gpt-3.5-turbo-16k";
```

Comment:
@stephentoub - `ChatName` or `ChatModel`? The existing code used both (ChatModel in code, ChatName in appsettings.json).

Reply:
If we're just talking about OpenAI, ChatModel is better. If we're also talking about Azure OpenAI, this is either a deployment name for Azure OpenAI or a model name for OpenAI.